src/modules/Dashboard/state/Applets/Applets.reducer.ts

```diff
@@ -7,6 +7,14 @@ import { AppletsSchema } from './Applets.schema';
 import { getApplets } from './Applets.thunk';
 import { state as initialState } from './Applets.state';
 
+export const reducers = {
+  deleteApplet: ({ applets }: AppletsSchema, action: PayloadAction<{ id: string }>): void => {
+    if (applets?.data) {
```

Comment:
please use early return, it's a good practice to avoid nesting

Reply:
no need here))